tests/rest/admin/test_user.py

```diff
@@ -344,7 +401,7 @@ def test_no_auth(self):
         self.render(request)
 
         self.assertEqual(401, int(channel.result["code"]), msg=channel.result["body"])
-        self.assertEqual("M_MISSING_TOKEN", channel.json_body["errcode"])
+        self.assertEqual(Codes.MISSING_TOKEN, channel.json_body["errcode"])
```

Comment:
I'm not entirely convinced that this is an improvement; in any case it doesn't seem relevant to the change at hand?

Reply:
Yes, you're right. It is not an improvement. It is the try to use consistently error codes as constant in all tests.

Reply:
if it is not an improvement, we should not do it.

it seems to me that the test is more thorough if we check the actual result. If we use the constant, we wouldn't detect a typo in the definition of the constant.